worker/uniter/operation/errors.go

```diff
@@ -19,23 +19,16 @@ var (
 	ErrCannotAcceptLeadership = errors.New("cannot accept leadership")
 )
 
-type deployConflictError struct {
+type DeployConflictError struct {
```

Comment:
Why export this now?

Reply:
Because we are removing IsDeployConflictError in favour of using errors.As which is  the correct way to do it now.

Reply:
Do we need to use a ConstError here?